lib/core/public/standards.js

```diff
@@ -0,0 +1,18 @@
+import ariaAttrs from '../../standards/aria-attrs';
+import { clone } from '../utils';
+
+const standards = {
+	ariaAttrs
+};
+
+export function configureStandards(config) {
```

Comment:
I think this needs to do a deep copy. We don't want to lose props on objects. For example if you configure like this, I don't think that should cause "type" to get dropped.

```js
axe.configure({ standards: { ariaAttrs: { 'aria-disabled': { global: false } } } });
```